.github/actions/vtk-android-install-dep/action.yml

```diff
@@ -0,0 +1,59 @@
+name: 'Install VTK Android Dependency'
+description: 'Install VTK Android Dependency using cache when possible'
+inputs:
+  arch:
+    description: 'Android CPU architecture'
+    required: true
+  api_level:
+    description: 'Android API level'
+    required: false
+    default: '27'
+runs:
+  using: "composite"
+  steps:
+
+    - name: Recover VTK Short SHA from file
+      if: steps.cache-vtk.outputs.cache-hit != 'true'
+      working-directory: ${{github.workspace}}
+      shell: bash
+      run: echo "VTK_SHA=$(<./source/.github/actions/vtk_commit_sha)" >> $GITHUB_ENV
+
+    - name: Cache VTK
+      id: cache-vtk
+      uses: actions/cache@v2
+      with:
+        path: dependencies/vtk_build
+        key: vtk-${{env.VTK_SHA}}-3-android-${{inputs.api_level}}-${{inputs.arch}}
```

Comment:
` vtk-android-${{env.VTK_SHA}}-3-${{inputs.api_level}}-${{inputs.arch}}` ?

not critical anyway